packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -165,6 +258,21 @@ class CupertinoCheckbox extends StatefulWidget {
   /// [RoundedRectangleBorder] with a circular corner radius of 4.0.
   final OutlinedBorder? shape;
 
+  /// True if this checkbox wants to show an error state.
```

Comment:
Also, is there anywhere `isError` is used? Maybe I missed but I don't see `isError` used in any of the default values. If the purpose is for customization, then users don't need the special `isError` because they can just customize from the outside using `fillColor: isError ? color1 : color2`.

Reply:
True. It's an antipattern copied over from Material checkbox and should be removed.